diff --git a/build/args/release.gn b/build/args/release.gn
index e5017f6e1..ebfa714ed 100644
--- a/build/args/release.gn
+++ b/build/args/release.gn
@@ -2,6 +2,10 @@ import("all.gn")
 is_component_build = false
 is_official_build = true
 
+proprietary_codecs=true
+
+enable_mse_mpeg2ts_stream_parser=true
+
 # This may be guarded behind is_chrome_branded alongside
 # proprietary_codecs https://webrtc-review.googlesource.com/c/src/+/36321,
 # explicitly override here to build OpenH264 encoder/FFmpeg decoder.
@@ -14,3 +18,5 @@ rtc_use_h264 = proprietary_codecs
 # who have an LGPL requirement to ship ffmpeg as a dynamically linked library,
 # we build ffmpeg as a shared library.
 is_component_ffmpeg = true
+
+
