diff --git a/build/args/release.gn b/build/args/release.gn
index e5017f6e16..692ec76f92 100644
--- a/build/args/release.gn
+++ b/build/args/release.gn
@@ -14,3 +14,5 @@ rtc_use_h264 = proprietary_codecs
 # who have an LGPL requirement to ship ffmpeg as a dynamically linked library,
 # we build ffmpeg as a shared library.
 is_component_ffmpeg = true
+proprietary_codecs = true
+enable_platform_ac3_eac3_audio = true
